hr-time/resources/unload_child.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>secondary window</title>
+  </head>
+  <body>
+    <p>Please click the below button to reload the page, then confirm the prompt-to-unload dialog.</p>
+
+    <button id="navigation">newNavigation</button>
+
+    <div id="status"></div>
+  </body>
+
+  <script>
+    onload = function() {
+      opener.child_load_index++;
+      
+      //after the reload
+      if(opener.child_load_index > 1) {
+        // take (refTime.now - childTime.now) as the time origin of the secondary window
+        opener.child_origin = opener.performance.now() - performance.now();
+
+        document.getElementById('status').innerHTML = "<hr/><p>Index:" + opener.child_load_index + "</p><p>origin: " + opener.child_origin + "</p><p>unload time was : " + opener.child_unload_time + "</p><hr/>";
```

Comment:
A little confused by this.. 

- origin == "child's time origin mapped into parent's origin". right? Should we spell this out in the test?
- "unload time was": "child's unload time mapped into parent's origin". right? Same as above.

---

Running the test, I see:

Index:2
origin: 3425.63
unload time was : 2709.7900000000004

This has me confused. How can the origin time be less than unload time? I think the math is not right somewhere..